Model/Schema.php

```diff
@@ -110,7 +110,7 @@ private function prepareProductsData(Order $order)
                                     'upc'          => $product->getUpc(),
                                     'isbn'         => $product->getIsbn(),
                                     'mpn'          => $product->getMpn(),
-                                    'brand'        => $product->getBrand(),
```

Comment:
why:
$product->getBrand() ? $product->getAttributeText('brand') : null,
and not 
$product->getBrand() ? $product->getBrand() : $product->getAttributeText('brand')

Reply:
will we be able to pull also the brand if the spec brand will be in text box and not select list?